### PR TITLE
Remove _gtk_builder_get_widget

### PR DESCRIPTION
**Pull request by tamplan**
change "gtk_builder_get_object (builder" to "GET_WIDGET ("
convert some tabs to spaces

### CODE REVIEW

**Comment by raveit65** [comment]
What is the advantage of this PR?
Does it fix an bug?
Is this really needed?

**Comment by tamplan** [comment]
> What is the advantage of this PR?
> Does it fix an bug?
> Is this really needed?

Really needed no, same as https://github.com/mate-desktop/mate-utils/pull/263 sure.

File: capplets/common/capplet-util.h [hunk 1]
```diff
@@ -33,6 +33,7 @@
 /* Retrieve a widget from the UI object */
 
 #define WID(s) GTK_WIDGET (gtk_builder_get_object (dialog, s))
+#define GET_WIDGET(x) (GTK_WIDGET (gtk_builder_get_object (builder, (x))))
```
Comment (rbuj) [inline capplets/common/capplet-util.h:36]: Move `#define GET_WIDGET...` at the top of:
- capplets/display/xrandr-capplet.c
- capplets/keybindings/mate-keybinding-properties.c
- capplets/network/mate-network-properties.c

File: capplets/network/mate-network-properties.c [hunk 2]
```diff
@@ -478,9 +470,9 @@ main (int argc, char **argv)
 	socks_proxy_settings = g_settings_new (SOCKS_PROXY_SCHEMA);
 
 	setup_dialog (builder);
-	widget = _gtk_builder_get_widget (builder, "network_dialog");
+	widget = GET_WIDGET ("network_dialog");
 
-        GtkNotebook* nb = GTK_NOTEBOOK (_gtk_builder_get_widget (builder, "notebook1"));
+        GtkNotebook* nb = GTK_NOTEBOOK (GET_WIDGET ("notebook1"));
```
Comment (rbuj) [inline capplets/network/mate-network-properties.c:475]: GtkNotebook* nb = GTK_NOTEBOOK (gtk_builder_get_object ("notebook1"));

File: capplets/network/mate-network-properties.c [hunk 3]
```diff
@@ -392,61 +384,61 @@ setup_dialog (GtkBuilder *builder)
...
+    g_settings_bind (socks_proxy_settings, SOCKS_PROXY_HOST_KEY,
+                     gtk_builder_get_object (builder, "socks_host_entry"), "text",
+                     G_SETTINGS_BIND_DEFAULT);
 
 	/* Autoconfiguration */
-	g_settings_bind (proxy_settings, PROXY_AUTOCONFIG_URL_KEY,
-			gtk_builder_get_object (builder, "autoconfig_entry"), "text",
-			G_SETTINGS_BIND_DEFAULT);
+    g_settings_bind (proxy_settings, PROXY_AUTOCONFIG_URL_KEY,
+                     gtk_builder_get_object (builder, autoconfig_entry"), "text",
```
Comment (rbuj) [inline capplets/network/mate-network-properties.c:424]: build error: missing terminating " character

Comment (rbuj) [reply capplets/network/mate-network-properties.c:424]:
```patch
--- a/capplets/network/mate-network-properties.c
+++ b/capplets/network/mate-network-properties.c
@@ -421,7 +421,7 @@ setup_dialog (GtkBuilder *builder)
 
        /* Autoconfiguration */
     g_settings_bind (proxy_settings, PROXY_AUTOCONFIG_URL_KEY,
-                     gtk_builder_get_object (builder, autoconfig_entry"), "text",
+                     gtk_builder_get_object (builder, "autoconfig_entry"), "text",
                      G_SETTINGS_BIND_DEFAULT);
 
     g_signal_connect (gtk_builder_get_object (builder, "network_dialog"),
```

File: capplets/network/mate-network-properties.c (hunk 2 above)
Comment (rbuj) [reply capplets/network/mate-network-properties.c]: GtkNotebook* nb = GTK_NOTEBOOK (gtk_builder_get_object (builder, "notebook1"));

Comment (tamplan) [reply capplets/network/mate-network-properties.c]: Done.

File: capplets/network/mate-network-properties.c [hunk 4]
```diff
@@ -392,61 +386,61 @@ setup_dialog (GtkBuilder *builder)
...
-			  "clicked", G_CALLBACK (cb_remove_url), builder);
+    config_treeview (GTK_TREE_VIEW (gtk_builder_get_object (builder, "treeview_ignore_host")), model);
+
+    g_signal_connect (gtk_builder_get_object (builder, "button_add_url"),
+                      "clicked", G_CALLBACK (cb_add_url), builder);
+    g_signal_connect (gtk_builder_get_object (builder, "entry_url"),
+                      "activate", G_CALLBACK (cb_add_url), builder);
+    g_signal_connect (gtk_builder_get_object (builder, "button_remove_url"),
+                      "clicked", G_CALLBACK (cb_remove_url), builder);
 }
```
Comment (rbuj) [inline capplets/network/mate-network-properties.c:444]: Do not fix the indent as it obfuscates the real change, and edit only the line : `GET_WIDGET ("network_dialog"));`.

**Comment by tamplan** [comment]
Ok, i delete this PR because of indentation, i'll make another one letter.

Thanks a lot.

**Pull request closed without merging**